File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Xuất tất cả các số nguyên tố/XUATSNT.py
# Hàm kiểm tra số nguyên tố
def is_prime(x):
    if x < 2:
        return False
    for i in range(2, x // 2 + 1):
        if x % i == 0:
            return False
    return True

# Nhập giá trị n
n = int(input())
if n < 2:
    print(-1)
else:
    # In ra các số nguyên tố từ 2 đến n
    for i in range(2, n + 1):
        if is_prime(i):
            print(i, end=" ")

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Xuất tất cả các số nguyên tố/XUATSNT.cs
using System;

namespace SNT
{
    public class Program
    {
        // Hàm kiểm tra số nguyên tố
        static bool IsPrime(int x)
        {
            if (x < 2)
                return false;
            for (int i = 2; i <= x / 2; i++)
                if (x % i == 0)
                    return false;
            return true;
        }

        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            if (n < 2)
            {
                Console.WriteLine("-1");
                return;
            }

            // In ra các số nguyên tố từ 2 đến n
            for (int i = 2; i <= n; i++)
            {
                if (IsPrime(i))
                {
                    Console.Write(i + " ");
                }
            }
        }
    }
}
